apps/amp-als/dataset-service/src/main/java/org/sagebionetworks/amp/als/dataset/service/api/DatasetApiDelegate.java

```diff
@@ -35,7 +35,7 @@ default Optional<NativeWebRequest> getRequest() {
    *         or The request cannot be fulfilled due to an unexpected server error (status code 500)
    * @see DatasetApi#getDataset
    */
-  default ResponseEntity<DatasetDto> getDataset(Long datasetId) {
+  default ResponseEntity<?> getDataset(Long datasetId) {
```

Comment:
Workaround to support returning both JSON and JSON-LD payloads (JSON-LD support will be added in an upcoming PR).
Note: This file is overwritten whenever the OpenAPI generator is run. A more robust solution would be to use a custom template. Additionally, the use of `?` should be replaced with a more descriptive type.